src/components/elements/upload/fileSummary/FileUploadSummary.tsx

```diff
@@ -0,0 +1,38 @@
+import { Typography } from '@mui/material';
+import React, { useMemo } from 'react';
+import FileSummary, {
+  FileSummaryProps,
+} from '@/components/elements/upload/fileSummary/FileSummary';
+import theme from '@/config/theme';
+
+// Wrapper around FileSummary for representing a File, aka a file that has just now been uploaded to Active Storage but doesn't yet exist as a File record in our db.
+const FileUploadSummary: React.FC<{
```

Comment:
Suggest giving this component a name that indicates that the file hasn't been uploaded yet- maybe `StagedFileSummary` or `UnsavedFileSummary`? At least to me it wasn't obvious the difference between FileRecord and FileUpload. `SavedFileSummary`/`UnsavedFileSummary` may be clearer